src/Robin/Ntlm/Crypt/Random/OpenSslRandomByteGenerator.php

```diff
@@ -15,7 +15,7 @@
  * "openssl" extension.
  *
  * @link http://php.net/openssl
- * @deprectated NOTE! This implementation is deprecated, as it's been found to
+ * @deprecated Insecure! This implementation is deprecated, as it's been found to
```

Comment:
Can you explain why a NOTE or TODO is needed within a deprecation tag? I've never seen that. I was intending to make this comment on the original PR had there been other changes to make. 

The deprecation tag explains that it's deprecated and why its deprecated. If a further todo is needed, such as `@todo Remove due to insecurity`, that should go in the main docblock.

Otherwise you could make the case that every `@deprecation` tag every warrants a `NOTE:` first. It's just a bit redundant.